shared/wallets/container.js

```diff
@@ -1,26 +1,27 @@
 // @flow
 import Wallets from '.'
 import * as WalletsGen from '../actions/wallets-gen'
-import {connect, type TypedState, type Dispatch, isMobile} from '../util/container'
+import {compose, connect, lifecycle, type TypedState, type Dispatch, isMobile} from '../util/container'
 import {HeaderHoc} from '../common-adapters'
 
-const mapStateToProps = (state: TypedState) => {
-  const {hello} = state.wallets
-  return {hello}
-}
+const mapStateToProps = (state: TypedState) => ({})
 
 const mapDispatchToProps = (dispatch: Dispatch, {navigateUp}) => ({
   onBack: () => dispatch(navigateUp()),
-  refresh: () => dispatch(WalletsGen.createWalletsRefresh()),
+  refresh: () => dispatch(WalletsGen.createLoadEverything()),
 })
 
 const mergeProps = (stateProps, dispatchProps) => ({
-  hello: stateProps.hello,
   onBack: dispatchProps.onBack,
   refresh: dispatchProps.refresh,
   title: 'Wallets',
 })
 
-export default connect(mapStateToProps, mapDispatchToProps, mergeProps)(
-  isMobile ? HeaderHoc(Wallets) : Wallets
-)
+export default compose(
+  connect(mapStateToProps, mapDispatchToProps, mergeProps),
+  lifecycle({
```

Comment:
we can leave this as its temp but fyi we're avoiding lifecycle as its very untyped